packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -400,6 +406,51 @@ describe('Overhead controller', () => {
 
           axios.get(`http://localhost:${serverConfig.port}${FIRST_REQUEST}`).then().catch(done)
         })
+
+        it('should add _dd.iast.enabled tag even when no vulnerability is detected', (done) => {
+          const config = new Config({
+            experimental: {
+              iast: {
+                enabled: true,
+                requestSampling: 100,
+                maxConcurrentRequests: 1
+              }
+            }
+          })
+          iast.enable(config)
+
+          const handler = function (traces) {
+            try {
+              for (let i = 0; i < traces.length; i++) {
+                for (let j = 0; j < traces[i].length; j++) {
+                  const trace = traces[i][j]
+                  if (trace.type === 'web') {
+                    const url = trace.meta['http.url']
+                    if (url.includes(SECURE_REQUEST)) {
+                      expect(trace.meta['_dd.iast.json']).to.be.undefined
+                      expect(trace.meta['_dd.iast.enabled']).eq('1')
+                    }
+                    done()
```

Comment:
Shouldn't this `done()` be into the `if (url.includes(SECURE_REQUEST)) {`?
If url does not include `SECURE_REQUEST` the test will be OK  instead of FAIL

Reply:
Yes.